packages/lodestar/src/network/nodejs/util.ts

```diff
@@ -31,11 +31,13 @@ export async function loadPeerIdFromJsonFile(path: string): Promise<PeerId> {
  * @param peerIdOrPromise Create an instance of NodejsNode asynchronously
  * @param network
  */
-export async function createNodeJsLibp2p(peerIdOrPromise: PeerId | Promise<PeerId>, 
-  network: Partial<INetworkOptions> = {}): Promise<LibP2p> {
+export async function createNodeJsLibp2p(
+  peerIdOrPromise: PeerId | Promise<PeerId>,
+  network: Partial<INetworkOptions> = {}
+): Promise<LibP2p> {
   const peerId = await Promise.resolve(peerIdOrPromise);
   const multiaddrs = network.multiaddrs || defaults.multiaddrs;
   const bootnodes = network.bootnodes || defaults.bootnodes;
   const peerInfo = await initializePeerInfo(peerId, multiaddrs);
-  return new NodejsNode({peerInfo, bootnodes: bootnodes});
-}
\ No newline at end of file
+  return new NodejsNode({peerInfo, bootnodes: bootnodes, discv5: network.discv5});
```

Comment:
Just a small note: we'll need to put in `ENRForkID` as spec 0.11 PR https://github.com/ethereum/eth2.0-specs/pull/1652/files

Reply:
:+1: yeah, the enr should be available through `libp2p._discovery.get("discv5").enr`.

I'm thinking of doing this in additional PRs.